modules/wp-cli/library.php

```diff
@@ -93,9 +99,17 @@ public function import( $args ) {
 			\WP_CLI::error( $imported_items->get_error_message() );
 		}
 
-		\WP_CLI::success( count( $imported_items ) . ' item(s) has been imported.' );
-	}
+		foreach ( $imported_items as $item ) {
+			$imported_items_ids[] = $item['template_id'];
+		}
+		$imported_items_ids = implode( ',', $imported_items_ids );
 
+		if ( 'ids' === $return_type ) {
+			\WP_CLI::line( $imported_items_ids );
```

Comment:
```suggestion
			\WP_CLI::success( $imported_items_ids );
```

Reply:
I'm sending the ids here, with the "Success: " in the response string it will be difficult to fetch those ids.